JustSaying/JustSayingFluently.cs

```diff
@@ -28,7 +28,7 @@ public class JustSayingFluently : ISubscriberIntoQueue, IHaveFulfilledSubscripti
         private readonly ILogger _log;
         private readonly IVerifyAmazonQueues _amazonQueueCreator;
         private readonly IAwsClientFactoryProxy _awsClientFactoryProxy;
-        protected IAmJustSaying Bus { get; }
+        protected IAmJustSaying Bus { get; private set; }
         private SqsReadConfiguration _subscriptionConfig = new SqsReadConfiguration(SubscriptionType.ToTopic);

```

Comment:
Could we get away with internal and use InternalsVisibleTo?

Reply:
If the test isn't super important and just belt-and-braces, we should remove it. It's a self-documented cludge, so if it's not adding much value we should just 🗑 it if it's causing us trouble here.

Reply:
The test looks like it's a base class for a lot of things :(

Reply:
Bah. A super-cludge.

Reply:
This is the theme in general with a lot of the tests

Reply:
Once #426 gets merged I'll see if I can get time to flex my ex-QA muscles and start slowly re-writing things for #380...

Reply:
@slang25  apprently not, setting it to `protected IAmJustSaying Bus { get; internal set; }` produces

```
JustSayingFluently.cs(31,53,31,56): 
error CS0273: 
The accessibility modifier of the 'JustSayingFluently.Bus.set' accessor 
must be more restrictive than the property or indexer 'JustSayingFluently.Bus'
```

Reply:
`protected internal`?

Reply:
Same.

Reply:
I meant for the whole property, not just the setter.

Reply:
Well TIL. https://stackoverflow.com/a/1063920/2225808
You could have a method that sets it, and is internal. 🤔 🤢

Reply:
Yeah, `protected internal` for the property is what we want